# awesome-single-cell-rna-seq
## Data correction

| Title                                                        | Team/Main author | Venue and year | Github                                            | Key Description |
| ------------------------------------------------------------ | ---------------- | -------------- | ------------------------------------------------- | --------------- |
| [Deep generative modeling for single-cell transcriptomics](https://www.nature.com/articles/s41592-018-0229-2) (scVI) <code class="language-plaintext highlighter-rouge">Nature</code> | Romain Lopez     | 2018           | [url](https://github.com/scverse/scvi-tools)      |                 |
| [BERMUDA: a novel deep transfer learning method for single-cell RNA sequencing batch correction reveals hidden high-resolution cellular subtypes](https://link.springer.com/article/10.1186/s13059-019-1764-6) (BERMUDA) `Genome biology` | Tongxin Wang     | 2019           | [url](https://github.com/txWang/BERMUDA)          |                 |
| [Exploring single-cell data with deep multitasking neural networks](https://www.nature.com/articles/s41592-019-0576-7) (SAUCIE) `Nature` | Matthew Amodio   | 2019           | [url](https://github.com/KrishnaswamyLab/SAUCIE/) |                 |
| [Deep learning enables accurate clustering with batch effect removal in single-cell RNA-seq analysis](https://www.nature.com/articles/s41467-020-15851-3) (DESC) `Nature` | Xiangjie Li      | 2020           | [url](https://github.com/eleozzr/desc)            |                 |

## Imputation

| Title                                                        | Team/Main author     | Venue and year | Github                                                | Key Description |
| ------------------------------------------------------------ | -------------------- | -------------- | ----------------------------------------------------- | --------------- |
| [AutoImpute: Autoencoder based imputation of single-cell RNA-seq data](https://www.nature.com/articles/s41598-018-34688-x) (AutoImpute) `Nature` | Divyanshu Talwar     | 2018           | [url](https://github.com/divyanshu-talwar/AutoImpute) |                 |
| [Single-cell RNA-seq denoising using a deep count autoencoder](https://www.nature.com/articles/s41467-018-07931-2) (DCA) `Nature` |                      | 2019           | [url](https://github.com/theislab/dca)                |                 |
| [DeepImpute: an accurate, fast, and scalable deep neural network method to impute single-cell RNA-seq data](https://link.springer.com/article/10.1186/s13059-019-1837-6) (DeepImpute) `Genome biology` | CÃ©dric Arisdakessian | 2019           | [url](https://github.com/lanagarmire/DeepImpute)      |                 |
| [deepMc: Deep Matrix Completion for Imputation of Single-Cell RNA-seq Data](https://www.liebertpub.com/doi/abs/10.1089/cmb.2019.0278) (Deepmc) `Journal of Computational Biology` | Aanchal Mongia       | 2019           | none                                                  |                 |
| [scIGANs: single-cell RNA-seq imputation using generative adversarial networks](https://academic.oup.com/nar/article/48/15/e85/5862684) (scIGANs) `Nucleic acids research` | Yungang Xu           | 2020           | [url](https://github.com/xuyungang/scIGANs)           |                 |
| [Imputing single-cell RNA-seq data by combining graph convolution and autoencoder neural networks](https://www.cell.com/iscience/fulltext/S2589-0042(21)00361-8) (GraphSCI) `Cell` | Jiahua Rao           | 2021           | [url](https://github.com/biomed-AI/GraphSCI)          |                 |

## Downstream clustering analysis

### Deep Graph Embedding Methods

| Title                                                        | Team/Main author | Venue and year | Github                                         | Key Description |
| ------------------------------------------------------------ | ---------------- | -------------- | ---------------------------------------------- | --------------- |
| [Clustering single-cell RNA-seq data with a model-based deep learning approach](https://www.nature.com/articles/s42256-019-0037-0) (scDeepCluster) <code class="language-plaintext highlighter-rouge">Nature</code> | Tian Tian        | 2019           | [url](https://github.com/ttgump/scDeepCluster) |                 |
| [Deep soft K-means clustering with self-training for single-cell RNA sequence data](https://academic.oup.com/nargab/article/2/2/lqaa039/5843803?login=false) (scziDesk) <code class="language-plaintext highlighter-rouge">NAR genomics and bioinformatics</code> | Liang Chen       | 2020           | [url](https://github.com/xuebaliang/scziDesk)  |                 |
| [Model-based deep embedding for constrained clustering analysis of single cell RNA-seq data](https://www.nature.com/articles/s41467-021-22008-3) (scDCC) <code class="language-plaintext highlighter-rouge">Nature</code> | Tian Tian        | 2021           | [url](https://github.com/ttgump/scDCC)         |                 |
| [scGMAI: a Gaussian mixture model for clustering single-cell RNA-Seq data based on deep autoencoder](https://academic.oup.com/bib/article/22/4/bbaa316/6029147?login=false) (scGMAI) <code class="language-plaintext highlighter-rouge">Briefings in Bioinformatics</code> | Bin Yu           | 2021           | [url](https://github.com/QUST-AIBBDRC/scGMAI/) |                 |
| [Deep structural clustering for single-cell RNA-seq data jointly through autoencoder and graph neural network](https://academic.oup.com/bib/article/23/2/bbac018/6529282) (scDSC) `Briefings in Bioinformatics` | Yanglan Gan      | 2022           | [url](https://github.com/DHUDBlab/scDSC)       |                 |

### Deep Graph Embedding Methods

| Title                                                        | Team/Main author | Venue and year | Github                                         | Key Description |
| ------------------------------------------------------------ | ---------------- | -------------- | ---------------------------------------------- | --------------- |
| [Accurately clustering single-cell RNA-seq data by capturing structural relations between cells through graph convolutional network](https://ieeexplore.ieee.org/abstract/document/9313569/) (GraphSCC) <code class="language-plaintext highlighter-rouge">BIBM</code> | Yuansong Zeng    | 2020           | [url](https://github.com/biomed-AI/GraphSCC)   |                 |
| [scGNN is a novel graph neural network framework for single-cell RNA-Seq analyses](https://www.nature.com/articles/s41467-021-22197-x) (scGNN) <code class="language-plaintext highlighter-rouge">Nature</code> | Juexin Wang      | 2021           | [url](https://github.com/juexinwang/scGNN)     |                 |
| [A topology-preserving dimensionality reduction method for single-cell RNA-seq data using graph autoencoder](https://www.nature.com/articles/s41598-021-99003-7) (scGAE) <code class="language-plaintext highlighter-rouge">Nature</code> | Zixiang Luo      | 2021           | [url](https://github.com/ZixiangLuo1161/scGAE) |                 |
| [ZINB-Based Graph Embedding Autoencoder for Single-Cell RNA-Seq Interpretations](https://ojs.aaai.org/index.php/AAAI/article/view/20392) (scTAG) <code class="language-plaintext highlighter-rouge">AAAI</code> | Zhuohan Yu       | 2022           | [url](https://github.com/Philyzh8/scTAG)       |                 |
| [Unsupervised Deep Embedded Fusion Representation of Single-Cell Transcriptomics](https://ojs.aaai.org/index.php/AAAI/article/view/25631) (scDEFR) <code class="language-plaintext highlighter-rouge">AAAI</code> | Yue Cheng        | 2023           | none                                           |                 |
| [Exploring the Interaction between Local and Global Latent Configurations for Clustering Single-Cell RNA-Seq: A Unified Perspective](https://ojs.aaai.org/index.php/AAAI/article/view/26107) (scTPF) <code class="language-plaintext highlighter-rouge">AAAI</code> | Nairouz Mrabah   | 2023           | [url](https://github.com/MMAMAR/scTPF)         |                 |
| [Generalized Cell Type Annotation and Discovery for Single-Cell RNA-Seq Data](https://ojs.aaai.org/index.php/AAAI/article/view/25672) (scGAD) <code class="language-plaintext highlighter-rouge">AAAI</code> | Yuyao Zhai       | 2023           | none                                           |                 |
| [Realistic Cell Type Annotation and Discovery for Single-cell RNA-seq Data](https://www.ijcai.org/proceedings/2023/552) (scPOT) <code class="language-plaintext highlighter-rouge">IJCAI</code> | Yuyao Zhai       | 2023           | none                                           |                 |
| [Toward Convex Manifolds: A Geometric Perspective for Deep Graph Clustering of Single-cell RNA-seq Data](https://www.ijcai.org/proceedings/2023/0540.pdf) (scTCM) <code class="language-plaintext highlighter-rouge">IJCAI</code> | Nairouz Mrabah   | 2023           | [url](https://github.com/MMAMAR/scTConvexMan)  |                 |
| [xTrimoGene: An Efficient and Scalable Representation Learner for Single-Cell RNA-Seq Data](https://papers.nips.cc/paper_files/paper/2023/hash/db68f1c25678f72561ab7c97ce15d912-Abstract-Conference.html) (xTrimoGene) <code class="language-plaintext highlighter-rouge">NeurIPS</code> | Jing Gong        | 2023           | none                                           |                 |
| [Unsupervised Gene-Cell Collective Representation Learning with Optimal Transport](https://ojs.aaai.org/index.php/AAAI/article/view/27789) (scGCOT) <code class="language-plaintext highlighter-rouge">AAAI</code> | Jixiang Yu       | 2024           | none                                           |                 |

